src/raster/r.damflood/SWE.h

```diff
@@ -1,27 +1,26 @@
 float velocita_breccia(int i, double h);
```

Comment:
Is _velocita_breccia_ the velocity of breach, break, or another concept?

Reply:
deepl also suggest gap, hole